Doc/glossary.rst

```diff
@@ -320,6 +320,11 @@ Glossary
       A module written in C or C++, using Python's C API to interact with the
       core and with user code.
 
+   f-string
+      String literals prefixed with f or F are commonly called "f-strings"
+      which is short for :dfn:`formatted string literal`.  See also
```

Comment:
This isn't the definition of "formatted string literal" (`:dfn:` is used for formatting term definitions). The reference "See also f-strings" looks self-recursive.

I think it would be better to write ` which is short for :ref:'formatted string literal <f-strings>' `.